pkg/etcd/etcd.go

```diff
@@ -70,6 +70,13 @@ const (
 	defaultKeepAliveTimeout = 10 * time.Second
 
 	maxBackupRetention = 5
+
+	etcdStatusType = v1.NodeConditionType("EtcdIsVoter")
+
+	unjoinedStatus  MemberStatus = "unjoined"
+	unhealthyStatus MemberStatus = "unhealthy"
+	learnerStatus   MemberStatus = "learner"
+	voterStatus     MemberStatus = "voter"
```

Comment:
If the type is exported, the consts should be exported too. Either change the type to `memberStatus`, or export these.

If we export them, I would probably style them them as `StatusUnjoined`, `StatusUnhealthy`, and so on - see the convention for consts at https://pkg.go.dev/net/http#pkg-constants